src/main/java/org/tinyradius/dictionary/DefaultDictionary.java

```diff
@@ -26,29 +26,33 @@
 	public static Dictionary getDefaultDictionary() {
 		return instance;
 	}
-	
+
 	/**
 	 * Make constructor private so that a DefaultDictionary
-	 * cannot be constructed by other classes. 
+	 * cannot be constructed by other classes.
 	 */
 	private DefaultDictionary() {
 	}
-	
+
 	private static final String DICTIONARY_RESOURCE = "org/tinyradius/dictionary/default_dictionary";
 	private static DefaultDictionary instance = null;
-	
+
 	/**
 	 * Creates the singleton instance of this object
 	 * and parses the classpath ressource.
 	 */
 	static {
 		try {
 			instance = new DefaultDictionary();
-    		InputStream source = DefaultDictionary.class.getClassLoader().getResourceAsStream(DICTIONARY_RESOURCE);
+			ClassLoader classLoader = DefaultDictionary.class.getClassLoader();
+			InputStream source = classLoader.getResourceAsStream("default_dictionary");
+			if (source == null) {
+				source = classLoader.getResourceAsStream(DICTIONARY_RESOURCE);
+			}
 			DictionaryParser.parseDictionary(source, instance);
```

Comment:
Isn't this already supported if you place the file in `org/tinyradius/dictionary/default_dictionary` location?  Is this change to allow `default_dictionary` at the root of the classpath?

Reply:
Library user may not like the idea of creating the org/tinyradius/dictionary/default_dictionary dir. If someone (like me) would like to make use the library in an application server (e.g. jboss or apache) he would like to use existing application server directories dedicated for keeping the config file (like e.g. $JBSOSS_HOME/server/$PROFILE/conf dir) to put the "default_dictionary" file there without a need to  create org/tinyradius/dictionary/default_dictionary  subdirectories there.
My pull request makes it possible.

Reply:
What if some other jar or other directory in classpath has a file called "default_dictionary" but for something else?  It would lead to obscure failure and sometimes is difficult to fix.

Like I said, I consider a bad practice to place a property file at the top level unless the name is unique enough.  I'd be ok with having the file named `tinyradius_dictionary`.

Reply:
Let's use the tinyradius_dictionary name while trying to load from root dir.
It this fine for you? If sot, then we can correct the code.